features/step_definitions/kata.rb

```diff
@@ -1,229 +1,111 @@
-  pod_runtime_class_name = pod('example-fedora-kata').runtime_class_name
-  if pod_runtime_class_name != 'kata'
-    raise "Pod's runtimeclass name #{pod_runtime_class_name} should be `kata`"
-  end
-
-  logger.info("Checking for running process `qemu` with the pod...")
-  # 2. check there's a process with the pod name `qemu`
+Given /^catalogsource #{QUOTED} exists in #{QUOTED} namespace$/ do |catalog_source_name, project_name|
   step %Q/I switch to cluster admin pseudo user/
-  node_cmd = "ps aux | grep qemu"
-  @result = node(pod.node_name).host.exec_admin(node_cmd)
-  raise "No qemu process detected inside pod node" unless @result[:response].include? 'qemu'
+  project(project_name)
+  raise "Failed to create catalog source" unless catalog_source("#{catalog_source_name}").exists?
+  logger.info("Catalog source #{catalog_source_name} exists")
 end
 
-Given /^there is a catalogsource for kata container$/ do
+When /^i install sandboxed-operator in #{QUOTED} namespace$/ do |kata_ns|
+  step %Q/I store master major version in the :master_version clipboard/
   step %Q/I switch to cluster admin pseudo user/
-  step %Q/I use the "openshift-marketplace" project/
-  step %Q|I obtain test data file "kata/catalogsource.yaml"|
-  user.cli_exec(:apply, f: 'catalogsource.yaml')
-  step %Q/a pod becomes ready with labels:/, table(%{
-    | olm.catalogSource=redhat-marketplace |
-  })
+  #step %Q|I obtain test data file "kata/release-#{cb.master_version}/deployment.yaml"|
+  #@result = user.cli_exec(:apply, f: "deployment.yaml")
+  step %Q|I obtain test data file "kata/namespace.yaml"|
+  @result = user.cli_exec(:apply, f: "namespace.yaml")
+  raise "Failed to install sandboxed-operator" unless @result[:success]
+  step %Q|I obtain test data file "kata/operatorgroup.yaml"|
+  @result = user.cli_exec(:apply, f: "operatorgroup.yaml")
+  raise "Failed to install sandboxed-operator" unless @result[:success]
+  step %Q|I obtain test data file "kata/subscription.yaml"|
+  @result = user.cli_exec(:apply, f: "subscription.yaml")
+  raise "Failed to install sandboxed-operator" unless @result[:success]
 end
 
-Given /^the kata-operator is installed(?: to #{OPT_QUOTED})? using OLM(?: (CLI|GUI))?$/ do | kata_ns, install_method |
-  ensure_admin_tagged
-  kata_ns ||= "openshift-sandboxed-containers-operator"
-  kata_config_name = "example-kataconfig"
-  step %Q/I store master major version in the :master_version clipboard/
-  raise "Kata operator OLM installation only supported for OCP >= 4.8" unless cb.master_version >= "4.8"
-  install_method ||= 'CLI'
-  # first check pre-req
-  step %Q/I switch to cluster admin pseudo user/
-  project('openshift-marketplace')
-  unless catalog_source('qe-app-registry').exists?
-    logger.info("Kata installation depends on `qe-app-registry`, which is missing in this cluster, calling step to create it...")
-    step %Q/I create "qe-app-registry" catalogsource for testing/
+Then /^operator should be up and running$/ do
```

Comment:
@valtair1891 I recommend make this step definition to be more specific.  The phrase `operator should be up and running` is too generic, so when a user reads this, there's no clear indication of what operator the step is referring to.

Reply:
I'm assuming, that since it's in kata.rb file, it's for Kata operator

Reply:
@valtair1891 step definitions are global, so this is visible to others regardless of where it's located.